newIDE/app/src/InstancesEditor/Grid.js

```diff
@@ -1,50 +1,48 @@
+// @flow
 import * as PIXI from 'pixi.js-legacy';
-import {
-  rgbColorToHexNumber,
-  // TODO: Add Flow typing in this file.
-  // , type RGBColor
-} from '../Utils/ColorTransformer';
-
-// Equal to #6868E8
-const DEFAULT_COLOR /*: RGBColor */ = {
-  r: 104,
-  g: 104,
-  b: 232,
-};
-
-export default class SelectionRectangle {
-  constructor({ viewPosition, options }) {
+import ViewPosition from './ViewPosition';
+import { type InstancesEditorSettings } from './InstancesEditorSettings';
+
+type Props = {|
+  viewPosition: ViewPosition,
+  instancesEditorSettings: InstancesEditorSettings,
+|};
+
+export default class Grid {
+  pixiGrid = new PIXI.Graphics();
+  instancesEditorSettings: InstancesEditorSettings;
+  viewPosition: ViewPosition;
+
+  constructor({ viewPosition, instancesEditorSettings }: Props) {
     this.viewPosition = viewPosition;
-    this.options = options;
+    this.instancesEditorSettings = instancesEditorSettings;
 
-    this.pixiGrid = new PIXI.Graphics();
     this.pixiGrid.hitArea = new PIXI.Rectangle(0, 0, 0, 0);
   }
 
-  setOptions(options) {
-    this.options = options;
+  setInstancesEditorSettings(instancesEditorSettings: InstancesEditorSettings) {
+    this.instancesEditorSettings = instancesEditorSettings;
   }
 
   getPixiObject() {
     return this.pixiGrid;
   }
 
   render() {
-    const options = this.options;
+    const instancesEditorSettings = this.instancesEditorSettings;
 
-    if (!options.grid) {
+    if (!instancesEditorSettings.grid) {
       this.pixiGrid.visible = false;
       return;
     }
 
-    const gridColor = options.gridColor || DEFAULT_COLOR;
-    const gridHex = rgbColorToHexNumber(gridColor);
-    const gridAlpha = gridColor.a || 1;
```

Comment:
not relevant any more ?

Reply:
Not anymore, but I've added a disableAlpha in the color picker! :)